Userland/Services/Taskbar/QuickLaunchWidget.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2021, Fabian Blatz <fabianblatz@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "QuickLaunchWidget.h"
+#include <LibConfig/Client.h>
+#include <LibCore/MimeData.h>
+#include <LibGUI/BoxLayout.h>
+#include <serenity.h>
+
+namespace Taskbar {
+
+constexpr const char* quick_launch = "QuickLaunch";
```

Comment:
Please use east-const here too.
Also, why not make a StringView?
```suggestion
constexpr auto quick_launch = "QuickLaunch"sv;
```

Reply:
Added the suggestion to the ammended commit :^)